crates/oxc_codegen/src/gen.rs

```diff
@@ -2976,6 +2978,8 @@ impl Gen for TSTypeParameterDeclaration<'_> {
             p.print_soft_newline();
             p.dedent();
             p.print_indent();
+        } else if p.is_jsx {
```

Comment:
Can you add a comment here for what source text this is for? i.e.  "<T, >() => {}

Reply:
Added